src/proposals/20250306/SparkEthereum_20250306.t.sol

```diff
@@ -14,7 +14,7 @@ import 'src/test-harness/SparkTestBase.sol';
 
 import { ChainIdUtils } from 'src/libraries/ChainId.sol';
 
-contract SparkEthereum_20250306Test is SparkEthereumTests {
+contract SparkEthereum_20250306Test is SparkEthereumTests, CommonSpellAssertions {
```

Comment:
Ah so this should actually be inheriting just the one `SparkTestBase` instead of `SparkEthereumTests`.